### PR TITLE
:arrow_up: ccnmtlsettings 0.1.4

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

File: dmt/settings_shared.py [hunk 1]
```diff
@@ -26,7 +26,6 @@
 ]
 
 TEMPLATE_CONTEXT_PROCESSORS += [  # noqa
-    'django.contrib.messages.context_processors.messages',
```
Comment (sdreher) [inline dmt/settings_shared.py]: What's the reasoning behind pulling this from ccnmtlsettings?

Comment (nikolas) [inline dmt/settings_shared.py]: the messages stuff is already included in ccnmtlsettings, so it's not necessary to have it in dmt's settings

Comment (nikolas) [inline dmt/settings_shared.py]: if you're suggesting the messaging stuff shouldn't be in ccnmtlsettings, we can talk about that

Comment (sdreher) [inline dmt/settings_shared.py]: Sorry, I misread this PR. Please ignore. I'll merge.

**Pull request merged**